eyelid-match-ops/benches/match-ops.rs

```diff
@@ -229,7 +230,13 @@ pub fn bench_inv(settings: &mut Criterion) {
 
     let rng = rand::thread_rng();
 
-    let p = sample::<FULL_RES_POLY_DEGREE>(rng);
+    let params = YasheParams {
+        t: 1024,
+        delta: 3.2,
+    };
+    let ctx: Yashe<2048> = Yashe::new(params);
```

Comment:
It could be FULL_RES_POLY_DEGREE. I didn't find any bugs. 

The idea is to have a set of parameters for each resolution (high, medium, low). So MAX_POLY_DEGREE would be different for each one.

Reply:
I think we can sort that out after ticket #59 gets merged. See the details here:
https://github.com/Inversed-Tech/eyelid/pull/69#discussion_r1583879731

(You might need to load more comments to see it.)